test/JDBC/jdbc_schedule

```diff
@@ -136,6 +133,9 @@ ignore#!#case_insensitive_collation-before-13-5-vu-cleanup
 ignore#!#jira-BABEL-3504-upgrade-vu-prepare
 ignore#!#jira-BABEL-3504-upgrade-vu-verify
 ignore#!#jira-BABEL-3504-upgrade-vu-cleanup
+ignore#!#babelfish_migration_mode-vu-prepare
```

Comment:
Let's add the test files in upgrade schedules like: https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/f63e34850a9ef49a09b77f1d68867e10e6334ca7#diff-68c81f7cdc618c86885bce9e06eac2667b0b3cd0285c5580d7e857050a3fbc48. Also,

Reply:
Updated in the latest commit